services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java

```diff
@@ -249,6 +249,34 @@
   @SerializedName("end_time")
   public abstract String endTime();
 
+  /**
+   * Two letter country code where the incident is located.
+   */
+  @Nullable
+  @SerializedName("iso_3166_1_alpha2")
+  public abstract String countryCodeAlpha2();
+
+  /**
+   * Three letter country code where the incident is located.
```

Comment:
Add a link to https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3.

Reply:
Done. /cc @LukasPaczos